cfme/tests/middleware/test_provider_crud.py

```diff
@@ -0,0 +1,8 @@
+from cfme.middleware.hawkular import HawkularProvider
+from cfme import login
+
+login.login_admin()
+
+hawkular = HawkularProvider('My Hawkular', 'livingontheedge.hawkular.org', '80')
+
+hawkular.create(cancel=False, validate_credentials=False)
```

Comment:
Noted. I should clarify test_provider_crud.py was a poor man unit test.  Will turn it to proper py.test